### DISCUSSION TITLE
Additional question for #1834

### DISCUSSION THREAD

**Discussion by vitoyucepi** [savonet/liquidsoap#2045] [General]
I asked a question in #1834, but didn't receive any answers.
Nevertheless this is a little cryptic to me, so I have to ask it once again.

Is it necessary to log messages like this?
```
2021/10/16 04:24:39 [request:3] Nonexistent file or ill-formed URI "???failed???"!
```
Maybe a `request` with broken URI has created. And the message is a result of this event.
As for me I think this message shouldn't be logged.

**Comment by toots** [comment]
Hi @vitoyucepi. I agree that this is confusing. This is a work around at the moment to prevent playlists from stopping after a reload has returned an empty playlist. This should be cleaned up, I'm converting this to an issue to keep track of it.
